generate-assets/src/lib.rs

```diff
@@ -649,6 +655,8 @@ fn get_bevy_crates(db: &CratesIoDb) -> Result<Vec<(String, String)>, rusqlite::E
                 AND repository = ?\
         ",
     )?;
+
+    // Required let and return due to bevy_crates_statement not living long enough.
```

Comment:
Is there a better solution that can keep `bevy_crates_statement` alive long enough?